src/PolyhedralGeometry/Serialization.jl

```diff
@@ -1,170 +1,131 @@
+using JSON
+
 ##############################################################################
 """
-    save_cone(Cone, String)
+    save(Cone, String)
 
 Save a cone to a file in JSON format. The first argument is the cone, the
 second argument is the filename.
 """
-function save_cone(C::Cone, filename::String)
+function save(C::Cone, filename::String)
    bigobject = pm_object(C)
    Polymake.save_bigobject(bigobject, filename)
 end
 
-"""
-    load_cone(String)
-
-Load a cone stored in JSON format, given the filename as input.
-"""
-function load_cone(filename::String)
-   bigobject = Polymake.load_bigobject(filename)
-   typename = Polymake.type_name(bigobject)
-   if typename != "Cone<Rational>"
-      throw(ArgumentError("Loaded object is not of polymake type Cone, it has type " * typename))
-   end
-   return Cone(bigobject)
-end
 
 ##############################################################################
 """
-    save_polyhedron(Cone, String)
+    save(Cone, String)
```

Comment:
This does not match the code below, nor does it fit into the conventions for docstrings. Should be like this (also applies elsewhere)
```suggestion
    save(P::Polyhedron, filename::String)
```

Reply:
Thanks